plugins/org.python.pydev/tests_analysis/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatementTest.java

```diff
@@ -0,0 +1,440 @@
+        checkImport("#from math import ceil\nfrom math import (sqrt   )    ", "from math import fmod",
+                "#from math import ceil\nfrom math import (sqrt, fmod   )    ",
+                80);
+
+        // test with "(xxxx)" and comma
+        checkImport("#from math import ceil\nfrom math import (sqrt,)", "from math import fmod",
+                "#from math import ceil\nfrom math import (sqrt, fmod,)",
+                80);
+
+        // test with "(xxxx)", spaces after and comma
+        checkImport("#from math import ceil\nfrom math import (sqrt,)    ", "from math import fmod",
+                "#from math import ceil\nfrom math import (sqrt, fmod,)    ",
+                80);
+
+        // -- END OF TESTS WITH AN IMPORT COMMENTED AND LINE BREAK -- //
+
+        // -- TESTS WITH COMMENTS INLINE -- //
+
+        // normal situation
+        checkImport("from math import ceil#foo", "from math import sqrt", "from math import ceil, sqrt#foo", 80);
+
+        // test with multiple spaces or tabs
+        checkImport("from math import ceil          #foo", "from math import sqrt",
+                "from math import ceil, sqrt          #foo", 80);
+
+        // test with comma
+        checkImport("from math import ceil,#foo", "from math import sqrt", "from math import ceil, sqrt,#foo", 80);
+
+        // test with "(xxxx)"
+        checkImport("from math import (ceil)#foo", "from math import sqrt", "from math import (ceil, sqrt)#foo", 80);
+
+        // test with "(xxxx)", spaces and tabs before comment and spaces and tabs before comment
+        checkImport("from math import (ceil)     #foo", "from math import sqrt",
+                "from math import (ceil, sqrt)     #foo",
+                80);
+
+        //!! test with "(xxxx)" and multiple spaces and tabs
+        checkImport("from math import (ceil       )#foo", "from math import sqrt",
+                "from math import (ceil, sqrt       )#foo",
+                80);
+
+        //!! test with "(xxxx)", spaces and tabs before comment and multiple spaces and tabs
+        checkImport("from math import (ceil       )   #foo", "from math import sqrt",
+                "from math import (ceil, sqrt       )   #foo",
+                80);
+
+        // test with "(xxxx)" and comma
+        checkImport("from math import (ceil,)#foo", "from math import sqrt", "from math import (ceil, sqrt,)#foo", 80);
+
+        // test with "(xxxx)", spaces and tabs before comment and comma
+        checkImport("from math import (ceil,)   #foo", "from math import sqrt", "from math import (ceil, sqrt,)   #foo",
+                80);
+
+        // -- END OF TESTS WITH COMMENTS INLINE -- //
+
+        // -- TESTS WITH STANDARD COLS VALUE AND SPACES AND TABS BEFORE -- //
+        /*
+        // normal situation with tabs and spaces before
+        checkImport("    from math import ceil", "from math import sqrt", "    from math import ceil, sqrt", 80);
```

Comment:
I see these are all commented out... either they should be uncommented to be run as usual or they should be removed.